src/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts

```diff
@@ -72,7 +72,7 @@ export class ExtensionsListView extends ViewletPanel {
 	}
```

Comment:
It is very unfortunate that you had to go into each subclass that overwrites the `renderHeaderTitle` so you change the heading. It would be great if this was somehow done just once. But @sandy081 can help more here

Reply:
@sandy081, I looked into changing https://github.com/Microsoft/vscode/blob/096d627d2b3738c4b4471ce03b55d673225339dd/src/vs/base/browser/ui/splitview/panelview.ts#L150,
But NVDA wasn't recognizing it as a header. Not sure why.

Reply:
Three classes (Call stack, Open editors, SCM) overriding `renderHeaderTitle` have custom titles with additional text which can be changed after. Current abstraction is good enough to render title headers in general. @isidorn @joaomoreno please check if you can call `super.renderHeaderTitle()` and add more stuff you want on top of it? Otherwise it's fine to leave it as it is.